### PR TITLE
HEEDLS-NONE- Add Tutorial ID to Order By

### PR DESCRIPTION
**Pull request by SChandarana**
Small bugfix to fix ordering of tutorials within the section page

### CODE REVIEW

File: DigitalLearningSolutions.Data/Services/SectionContentService.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ GROUP BY
                         AND Sections.SectionID = @sectionId
                         AND (Sections.ArchivedDate IS NULL)
                         AND (CustomisationTutorials.DiagStatus = 1 OR Customisations.IsAssessed = 1 OR CustomisationTutorials.Status = 1)
-                    ORDER BY Tutorials.OrderByNumber",
+                    ORDER BY Tutorials.OrderByNumber, Tutorials.TutorialID",
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Services/SectionContentService.cs:86]: Is there a test we can add for this? And this probably needs to be done on the initial menu page as well, for the sections? If so @benjimarshall it probably makes sense for you to do that as you're changing that page? 🙂

**Review by FridaTveit — COMMENTED** [review]
One question, otherwise looks good 🙂

File: DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs [hunk 2]
```diff
@@ -316,5 +316,64 @@ public void Get_section_content_should_return_content_if_only_tutorial_status_is
...
+
+            // All in section 216
+            // Tutorial: 927  OrderByNumber 34
+            // Tutorial: 928  OrderByNumber 35
+            // Tutorial: 929  OrderByNumber 35
+            // Tutorial: 930  OrderByNumber 36
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            // Then
+            var expectedSectionContent = new SectionContent(
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs:336]: Could maybe use some helper functions to create the section and tutorials here? As we only really care about the order, so only need to specify enough to make sure that's correct 🙂

**Review by FridaTveit — COMMENTED** [review]
One suggestion, otherwise looks good 🙂

File: DigitalLearningSolutions.Data/Services/SectionContentService.cs (hunk 1 above)
Comment (FridaTveit) [reply DigitalLearningSolutions.Data/Services/SectionContentService.cs:86]: @benjimarshall? 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

File: DigitalLearningSolutions.Data/Services/SectionContentService.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data/Services/SectionContentService.cs:86]: Yep I'll add that to HEEDLS-230. Sorry for missing these comments 🙂

Comment (FridaTveit) [reply DigitalLearningSolutions.Data/Services/SectionContentService.cs:86]: Great, thanks 🙂

**Pull request merged**